nucleus/common/mbeanserver/src/main/java/org/glassfish/admin/mbeanserver/RMIConnectorStarter.java

```diff
@@ -137,7 +139,7 @@ public RMIConnectorStarter(
         }
 
         final boolean ENABLED = true;
-        mBindToSingleIP = ENABLED && !(address.equals("0.0.0.0") || address.equals(""));
+        mBindToSingleIP = ENABLED && !(address.equals("0.0.0.0") || address.equals("*")|| address.equals("")); 
```

Comment:
Is host does not listens on 0.0.0.0 ? I tested with CurrPort tool.  Is host does not listens on 0.0.0.0 ? I tested with CurrPort tool.